src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriors.java

```diff
@@ -0,0 +1,224 @@
+    public static final String POSTERIOR_CALL_DIRECTORY_SHORT_NAME = "CCD";
+
+    public static final String SAMPLE_DIRECTORY_NAME_FULL_NAME = "sample-name-directory";
+    public static final String SAMPLE_DIRECTORY_NAME_SHORT_NAME = "SN";
+
+    private static final String COMMENT_PREFIX = "@";
+
+    @Argument(
+            doc = "List of paths to chunk directories in a sorted order",
+            fullName = POSTERIOR_CALL_DIRECTORY_FULL_NAME,
+            shortName = POSTERIOR_CALL_DIRECTORY_SHORT_NAME
+    )
+    private List<File> orderedChunkDirectoryList = null;
+
+    @Argument(
+            doc = "Name of the sample directory in the gCNV called chunks (it must be the same across all chunks)",
+            fullName = SAMPLE_DIRECTORY_NAME_FULL_NAME,
+            shortName = SAMPLE_DIRECTORY_NAME_SHORT_NAME
+    )
+    private String sampleDirectoryName = null;
+
+    @Argument(
+            doc = "Output VCF file",
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME
+    )
+    private File outputFile = null;
+
+    /**
+     * Sample name derived from the copy number posterior file
+     */
+    private String sampleName;
+
+    final String variantPrefix = "CNV";
+
+    @Override
+    public void traverse() {
+        if (orderedChunkDirectoryList.size() <= 0) {
+            throw new UserException.BadInput("You must specify at least one chunk directory");
+        }
+
+        final SimpleIntervalCollection firstChunkSimpleIntervalCollection =
+                new SimpleIntervalCollection(new File(getIntervalFileFromChunkDirectory(orderedChunkDirectoryList.get(0).getAbsolutePath()).getAbsolutePath()));
+        final SAMSequenceDictionary samSequenceDictionary = firstChunkSimpleIntervalCollection.getMetadata().getSequenceDictionary();
+        Triple<List<CopyNumberPosteriorLocatableRecord>, IntegerCopyNumberStateCollection, String> firstCopyNumberPosteriorFileInfo =
+                readChunkedPosteriorFileFromDirectory(orderedChunkDirectoryList.get(0), samSequenceDictionary);
+        final IntegerCopyNumberStateCollection copyNumberStateCollection = firstCopyNumberPosteriorFileInfo.getMiddle();
+        sampleName = firstCopyNumberPosteriorFileInfo.getRight();
+
+        if (firstCopyNumberPosteriorFileInfo.getMiddle().size() < 3) {
+            throw new UserException.BadInput("There should be at least 3 copy number states");
+        }
+
+        final VariantContextWriter outputWriter = GATKVariantContextUtils.createVCFWriter(outputFile, null, false);
+        final GCNVPostProcessor gcnvPostProcessor = new GCNVPostProcessor(outputWriter, copyNumberStateCollection,
+                sampleName, samSequenceDictionary);
+
+        //TODO pass the command line invocation string to the header composer method
+        gcnvPostProcessor.composeVariantContextHeader(null);
```

Comment:
I will add the header in the new release as, I need I am comparing VCF file as plain text files in the integration test and adding command line will break integration tests on travis. I will write a smarter VCF comparator.

Reply:
If you include the return value from GATKTool.getDefaultToolVCFHeaderLines when you create the header, then you can write the integration tests with `--add-output-vcf-command-line false`, which will suppress the command line output will and make file comparison easier.

Reply:
I believe it will still break my integration test at the moment as GATKTool.getDefaultToolVCFHeaderLines also outputs the local time the tool was run. I will make an issue for writing a smarter integration test and skip this for now.

Reply:
@asmirnov239 Adding `--add-output-vcf-command-line false` to your integration test command lines should suppress all of the command line output, and keep the output header invariant across runs (at least with respect to the default header lines). The integration tests for HaplotypeCaller, VQSR, SelectVariants, etc. all do this. If for some reason it doesn't work, please file a ticket. Thx.

Reply:
Oh great. Done

Reply:
Done